src/ocrd/processor/base.py

```diff
@@ -505,7 +506,7 @@ def process_workspace(self, workspace: Workspace) -> None:
                 # set up multitasking
                 max_workers = max(0, config.OCRD_MAX_PARALLEL_PAGES)
                 if self.max_workers > 0 and self.max_workers < config.OCRD_MAX_PARALLEL_PAGES:
-                    self._base_logger.info("limiting number of threads from %d to %d", max_workers, self.max_workers)
+                    self._base_logger.info("limiting number of workers from %d to %d", max_workers, self.max_workers)
```

Comment:
Should we also do `s,threads,workers,` in the description of `OCRD_MAX_PARALLEL_PAGES` config option?

Reply:
indeed!
see 217059db